sources/packages/forms/src/form-definitions/studentinformation.json

```diff
@@ -654,194 +835,813 @@
+                        "customPrivate": false,
+                        "json": "",
+                        "strictDateValidation": false,
+                        "multiple": false,
+                        "unique": false
+                      },
+                      "unique": false,
+                      "errorLabel": "",
+                      "errors": "",
+                      "key": "lastName",
+                      "tags": [],
+                      "properties": {},
+                      "conditional": {
+                        "show": null,
+                        "when": null,
+                        "eq": "",
+                        "json": ""
+                      },
+                      "customConditional": "",
+                      "logic": [],
+                      "attributes": {
+                        "data-cy": "lastName"
+                      },
+                      "overlay": {
+                        "style": "",
+                        "page": "",
+                        "left": "",
+                        "top": "",
+                        "width": "",
+                        "height": ""
+                      },
+                      "type": "textfield",
+                      "input": true,
+                      "hideOnChildrenHidden": false,
+                      "refreshOn": "",
+                      "dataGridLabel": false,
+                      "addons": [],
+                      "inputType": "text",
+                      "id": "e90x9q",
+                      "defaultValue": ""
+                    }
+                  ],
+                  "width": 6,
+                  "offset": 0,
+                  "push": 0,
+                  "pull": 0,
+                  "size": "md",
+                  "currentWidth": 6
+                },
+                {
+                  "components": [
+                    {
+                      "label": "Date of birth",
+                      "labelPosition": "top",
+                      "labelWidth": "",
+                      "labelMargin": "",
+                      "displayInTimezone": "viewer",
+                      "useLocaleSettings": false,
+                      "allowInput": true,
+                      "format": "yyyy-MM-dd",
+                      "placeholder": "",
+                      "description": "",
+                      "tooltip": "",
+                      "customClass": "",
+                      "tabindex": "",
+                      "hidden": false,
+                      "hideLabel": false,
+                      "autofocus": false,
+                      "disabled": false,
+                      "tableView": false,
+                      "modalEdit": false,
+                      "shortcutButtons": [],
+                      "enableDate": true,
+                      "enableMinDateInput": true,
+                      "datePicker": {
+                        "minDate": "moment().subtract(100, 'years')",
```

Comment:
😄 . What if some with of age 101 want to apply 😉

Reply:
There is no business requirement for this, hence I will remove it.

Reply:
Removed.